guide/english/natural-language-processing/index.md

```diff
@@ -31,6 +31,11 @@ Next task is to build a language model. Here we consider an assumption that the
 ## Further readings
 The field of NLP is huge. If you have read this far and have implemented the above, you have certainly loved this. Go ahead and read Jurafsky's book to learn some more new concepts. Remember, it's important to implement them as well.
 
+To get started with NLP with python, one need to download the nltk kit using:
+
+    pip install nltk
```

Comment:
Please specify which version of Python this is targeting. Also, provide install instructions for the other version, and add a reference link at the bottom of the guide.